// try{
//     console.log(x);
//   }  // we cannot use variable before dec it araise reference error.
// catch{
//     let x =4;
// console.log(x);}

let num = 8;
try{
num.toUpperCase();}
catch(error)
{
console.log(num);}